### PR TITLE
add mlp bias for llama models

### PR DESCRIPTION
**Pull request by mayank31398**
This adds bias support for MLP in Llama
Can we add this?
It would help a lot for some models we are developing
@ArthurZucker and @younesbelkada

This lets us re-use the llama model class for our models :) without adding another class for a new model.

### CODE REVIEW

**Comment by mayank31398** [comment]
@younesbelkada light ping again

**Review by ArthurZucker — COMMENTED** [review]
Hey! Though this is not necessarily against our transformers philosophy, we would need to have justification: meaning a new cool model *released* rather than a promise that it will be release!
This should be easy to have on the hub with `trust_remote_code=True` no?

**Review by ArthurZucker — APPROVED** [review]
ALright! Now that you have a new model coming this makes sense. Could you make sure the CIs pass?

File: src/transformers/models/llama/configuration_llama.py [hunk 1]
```diff
@@ -98,6 +98,8 @@ class LlamaConfig(PretrainedConfig):
             Whether to use a bias in the query, key, value and output projection layers during self-attention.
         attention_dropout (`float`, *optional*, defaults to 0.0):
             The dropout ratio for the attention probabilities.
+        mlp_bias (`bool`, defaults to `False`, *optional*, defaults to `False`):
```
Comment (younesbelkada) [inline src/transformers/models/llama/configuration_llama.py:101]:
```suggestion
        mlp_bias (`bool`, *optional*, defaults to `False`):
```

File: src/transformers/models/llama/configuration_llama.py [hunk 2]
```diff
@@ -98,6 +98,8 @@ class LlamaConfig(PretrainedConfig):
...
+            Whether to use a bias in up_proj, down_proj and gate_proj layers in MLP.
```
Comment (younesbelkada) [inline src/transformers/models/llama/configuration_llama.py:102]:
```suggestion
            Whether to use a bias in up_proj, down_proj and gate_proj layers in the MLP layers.
```

**Review by younesbelkada — APPROVED** [review]
Great work !

**Review by younesbelkada — APPROVED** [review]
Great, thanks !

**Comment by mayank31398** [comment]
Thanks everyone for the quick turnaround 🤗

**Pull request merged**